src/agoradatatools/great_expectations/gx/plugins/expectations/expect_column_nested_field_values_to_be_mostly_not_null.py

```diff
@@ -0,0 +1,329 @@
+                    },
+                    "out": {"success": True},
+                },
+                # Fails: 1 of 3 values is null, NOT satisfying the 0.9 non-null threshold.
+                {
+                    "title": "target_fail_threshold",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {
+                        "column": "a",
+                        "non_null_threshold": 0.9,
+                        "target_field": "targeted",
+                    },
+                    "out": {"success": False},
+                },
+                # Passes: 1 of 2 values is null, satisfying the 0.5 non-null threshold.
+                {
+                    "title": "target_half_null",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {
+                        "column": "b",
+                        "non_null_threshold": 0.5,
+                        "target_field": "targeted",
+                    },
+                    "out": {"success": True},
+                },
+                # Fails: 5 of 5 values are null, NOT satisfying the 0.1 non-null threshold.
+                {
+                    "title": "target_all_null",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {
+                        "column": "c",
+                        "non_null_threshold": 0.1,
+                        "target_field": "targeted",
+                    },
+                    "out": {"success": False},
+                },
+                # Fails: 2 of 2 values are null, NOT satisfying the 0.01 non-null threshold.
+                {
+                    "title": "null_and_empty_list",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {
+                        "column": "d",
+                        "non_null_threshold": 0.001,
+                        "target_field": "targeted",
+                    },
+                    "out": {"success": False},
+                },
+                # Success: 1 of 3 values is invalid, satisfying the non-null threshold.
+                {
+                    "title": "target_mix_with_null",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {
+                        "column": "e",
+                        "non_null_threshold": 0.6,
+                        "target_field": "targeted",
+                    },
+                    "out": {"success": True},
+                },
+            ],
+        }
+    ]
+
+    metric_dependencies = (METRIC_NAME,)
+    success_keys = ("non_null_threshold", "target_field")
+    default_kwarg_values = {}
+
+    def validate_configuration(
+        self, configuration: Optional[ExpectationConfiguration]
```

Comment:
`configuration` is marked as optional, but still requires that the argument is passed in since it is not defaulting to `None`. Does this need to be `configuration: Optional[ExpectationConfiguration] = None`?

Reply:
Just added: 
```
configuration: Optional[ExpectationConfiguration] = None
```